webapp/channels/src/components/widgets/inputs/input/input.tsx

```diff
@@ -263,11 +263,6 @@ const Input = React.forwardRef((
                             {'-'}{limitExceeded}
                         </span>
                     )}
-                    {isMinLengthError && (
```

Comment:
So, this input field is being used in a log of places, marketplace modal, system console,etc. We should not remove this error indicator universally unless we're sure if its needed in any place. I'd suggest adding a prop `showMinLengthError` and default it to true and set it to false only in channel settings modal's relevant inputs.

Reply:
this was never part of the specs, this was something I added because I thought that could be potentially useful in the future (https://github.com/mattermost/mattermost/pull/30338/files#diff-a0438aef599678b3128f793c3025c9123eb3091ace275397f3a528d895a09d55R266). I still think it can and I like your idea of keeping it with a flag, but turned off by default then.